boot/bootstate20.go

```diff
@@ -43,6 +43,11 @@ func newBootState20(typ snap.Type) bootState {
 
 type bootState20Modeenv struct {
 	modeenv *Modeenv
+
+	// the base_status to be written to the modeenv, stored separately to
+	// eliminate unnecessary writes to the modeenv when it's already in the
+	// state we want it in
+	commitBaseStatus string
```

Comment:
this moved field is slightly odd,  I don't have simple suggestions to avoid this though

Reply:
I mean we could just close this PR and leave it as is, it's not hurting anything right now